src/gridstack.js

```diff
@@ -1041,10 +1041,14 @@
         this._updateContainerHeight();
     };
 
-    GridStack.prototype.destroy = function() {
+    GridStack.prototype.destroy = function(detachGrid) {
         $(window).off('resize', this.onResizeHandler);
         this.disable();
-        this.container.remove();
+        if (typeof detachGrid != 'undefined' && !detachGrid) {
+            this.removeAll(true);
```

Comment:
Shouldn't this be `false`? Otherwise, when `detachGrid === false`, we won't remove the grid container, but we will remove all of the widgets.